ui/src/text_annotator.tsx

```diff
@@ -208,10 +243,8 @@ export
         return false
       },
       getMark = (text: S, idx: U, tag: S) => {
-        const color = tagColorMap.get(tag)
-        // Handle invalid tags entered by user
-        if (!color) return text
```

Comment:
Why was this removed?

Reply:
Probably bad rebase since handling of invalid tags was implemented here: https://github.com/h2oai/wave/pull/1435